storefront-reference-architecture-master/cartridges/app_storefront_custom_cartridge/cartridge/templates/default/templates.isml

```diff
@@ -0,0 +1,19 @@
+<isdecorate template="common/layout/page">
+
+<h1>hy<isprint value="${pdict.msg}" /></h1>
+
+<isscript>
+    let arr=[1,2,3];
```

Comment:
Do we need this array?

Reply:
No, this is not from the home work i will remove it